models/members.js

```diff
@@ -148,8 +148,31 @@ const fetchUsersWithRole = async (role) => {
   }
 }
 
+/**
+ * changes the role of a new user to member
+ * @param userId { String }: User id of user to be modified
+ * @return { Object }: whether moveToMember was successful or not and whether user is already a member or not
+ */
+
+const addArchiveRoleToMembers = async (userId) => {
+  try {
+    const userDoc = await userModel.doc(userId).get()
+    const user = userDoc.data()
+    if (user?.roles?.archivedMember) return { isArchived: true, addArchiveRole: false }
```

Comment:
Do we need `addArchiveRole` to be returned? Because we are not using it in the controller.

Reply:
Oh yes, we don't need it. I added for better understanding. Is there any downside of returning extra value?

Reply:
For personal use it's ok. But in production unnecessary code is not good.

Reply:
Yeah makes sense. Removed it.